sonic-psud/scripts/psud

```diff
@@ -57,6 +57,9 @@ PSU_INFO_VOLTAGE_MAX_TH_FIELD = 'voltage_max_threshold'
 PSU_INFO_VOLTAGE_MIN_TH_FIELD = 'voltage_min_threshold'
 PSU_INFO_CURRENT_FIELD = 'current'
 PSU_INFO_POWER_FIELD = 'power'
+PSU_INFO_POWER_OVERLOAD = 'power_overload'
+PSU_INFO_POWER_WARNING_THRESHOLD = 'power_warning_threshold'
```

Comment:
'warning' can be misleading here as no warning is raised if this threshold is crossed. can we rename it as 'warning-suppress' levle?

Reply:
Done. Changed to `power_warning_suppress_threshold`